client/main.go

```diff
@@ -465,7 +465,14 @@ func main() {
 					jitterHist.RecordValue(jitter)
 					globalJitterHist.RecordValue(jitter)
 				}
-				if *totalRequests > 0 && totalCount >= *totalRequests {
+				if *totalRequests > 0 && totalCount + 1 >= *totalRequests {
+					if resp.err != nil {
+						bad++
+						totalBad++
+					} else {
+						good++
+						totalGood++
+					}
```

Comment:
@Smileyman5 No problem at all -- thanks for your submission! Am really glad to know somebody else is also using this project and vetting it for correctness. Let us know if you run into any other issues!